curriculum/challenges/english/25-front-end-development/workshop-greeting-card/67333d70431f8a4b02596f3c.md

```diff
@@ -0,0 +1,151 @@
+---
+id: 67333d70431f8a4b02596f3c
+title: Step 26
+challengeType: 0
+dashedName: step-26
+---
+
+# --description--
+
+Add a selector that targets the `section` elements when hovered. Set the `transform` property to `skewX(10deg)`. 
```

Comment:
It doesn't like the `skewX(10deg)` value has been taught yet.

Add a short sentence before this explaining what it is with a short code snippet.